### DISCUSSION TITLE
help needed. Loading SVG via CSS in node_modules is not working in NextJS 🤔

### DISCUSSION THREAD

**Discussion by NisugaJ** [vercel/next.js#24922] [Help]
I expect your support from anyone reading this. Lots of thanks if you can join to fix [this issue](https://github.com/vercel/next.js/discussions/24856). Almost wasted many days trying to fix [this issue](https://github.com/vercel/next.js/discussions/24856).

**Comment by bobarros** [comment]
First, show us what you're trying to do, and not Webpack config. I know that you believe the problem is there, but you might be wrong.

The code importing the SVG, using the imported result, and also the vector content will be helpful to make it quick since you lost a lot of time already.

What are your limitations? 
It is a problem to edit the SVG if needed? What the goal of this file?
It is possible to change the way you're importing? It is possible to use a different file?
